apple-news.php

```diff
@@ -79,29 +71,20 @@ function apple_news_uninstall_wp_plugin() {
 require __DIR__ . '/includes/class-apple-news.php';
 require __DIR__ . '/admin/class-admin-apple-news.php';
 
-/**
- * Load plugin textdomain.
- *
- * @since 0.9.0
- */
-function apple_news_load_textdomain() {
-	load_plugin_textdomain( 'apple-news', false, __DIR__ . '/lang' );
-}
-add_action( 'plugins_loaded', 'apple_news_load_textdomain' );
-
 /**
  * Gets plugin data.
  * Used to provide generator info in the metadata class.
  *
- * @return array
- *
  * @since 1.0.4
+ *
+ * @param bool $translate Whether to translate the plugin data.
+ * @return array
  */
-function apple_news_get_plugin_data() {
+function apple_news_get_plugin_data( $translate = true ) {
```

Comment:
Allow this function to be used before `init`, where translation is not supported or necessary. See the related changed unit test as part of this pr.